darwin/exporter/formats/nifti.py

```diff
@@ -350,7 +351,6 @@ def populate_output_volumes_from_polygons(
                 plane,
                 frame_idx,
             )
-    return volume
```

Comment:
This return statement actually isn't needed since we don't use `volume` anywhere. The function instead modifies the value of `output_volumes`

Reply:
@JBWilkie Can you update the docs for this function too? Otherwise, this PR is good.